nautobot/extras/tables.py

```diff
@@ -904,18 +911,20 @@ def render_summary(self, record):
         """
         Define custom rendering for the summary column.
         """
-        return format_html(
-            """<label class="label label-default">{}</label>
-            <label class="label label-success">{}</label>
-            <label class="label label-info">{}</label>
-            <label class="label label-warning">{}</label>
-            <label class="label label-danger">{}</label>""",
-            record.debug_log_count,
-            record.success_log_count,
-            record.info_log_count,
-            record.warning_log_count,
-            record.error_log_count,
-        )
+        if get_settings_or_config("JOB_RESULTS_SUMMARY_FIELD_ENABLED"):
```

Comment:
Is this check needed? This column should be hidden (and therefore this function shouldn't even be called) if that setting is disabled, right?

(Edit: I don't see that logic on the view diffs at this time, but I think it would be appropriate to add?)

Reply:
Just debugged this and recognize it is not being called.